components/tooltip/index.css

```diff
@@ -13,17 +13,75 @@ governing permissions and limitations under the License.
 @import "../overlay/index.css";
 
 .spectrum-Tooltip {
-  --spectrum-tooltip-neutral-target-offset: 3px;
-  /* This variable needs to be updated in Spectrum DNA. Using the height variable so it scales to spectrum-large */
-  --spectrum-tooltip-neutral-tip-width: var(--spectrum-tooltip-neutral-tip-height);
+  --spectrum-tooltip-animation-duration: var(--spectrum-animation-duration-100);
+
+  /* animation distance is equal to spacing to source */
+  --spectrum-tooltip-animation-distance: var(--spectrum-spacing-75);
+
+  /* override if additional spacing to source is required */
+  --spectrum-tooltip-margin: 0px;
+
+  --spectrum-tooltip-height: var(--spectrum-component-height-75);
+  --spectrum-tooltip-max-inline-size: var(--spectrum-tooltip-maximum-width);
+  --spectrum-tooltip-border-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-tooltip-icon-width: var(--spectrum-workflow-icon-size-50);
+  --spectrum-tooltip-icon-height: var(--spectrum-workflow-icon-size-50);
+
+  --spectrum-tooltip-font-size: var(--spectrum-font-size-75);
+  --spectrum-tooltip-line-height: var(--spectrum-line-height-100);
+  --spectrum-tooltip-line-height-cjk: var(--spectrum-cjk-line-height-100);
```

Comment:
We haven't been super consistent or rigid with naming conventions, but this is something that we _probably_ should standardize around, generally.

That said, it looks like, in other migrated components, we've stuck to a `cjk` naming convention like: `--spectrum-tooltip-cjk-line-height`. Would you mind updating this custom property to conform to that?